src/js/core/methods/CardanoSignTransaction.js

```diff
@@ -143,7 +143,7 @@ export default class CardanoSignTransaction extends AbstractMethod {
         }
 
         params.outputs.map((output) => {
-            if (output.token_bundle) {
+            if (output.token_bundle.length > 0) {
```

Comment:
```suggestion
            if (output.token_bundle && output.token_bundle.length > 0) {
```
token_bundle is optional right?

Reply:
yes, but it gets translated to an emoty array above, but I agree for the sake of "fool proofness" better explicitly check for its presence